clis/dtcli/src/doc/dtcli_documentation.md

```diff
@@ -0,0 +1,89 @@
+# DTCLI
+
+A daemonized version of TCLI, which can be used in scripts.
+
+## How To Build DTCLI
+
+Currently DTCLI is implemented as a package of the libtelio project, just like TCLI, so it is build just like TCLI too:
+
+```shell
+cargo build -p dtcli
+```
+
+## How To Use DTCLI
+
+### Prerequisites
+
+DTCLI needs a user group called `nordvpn` to be present on the current machine to work. If you have nordvpn installed, the group probably already exists, but if it doesn't you can add it with `groupadd nordvpn`
```

Comment:
you probably also what to add line saying, 'then add user to it  usermod -aG nordvpn $USER, and relogin'

Reply:
Done